iscsi/get.go

```diff
@@ -137,13 +137,14 @@ func ReadWriteOPS(iqnPath string, tpgt string, lun string) (readmb uint64,
 
 // GetFileioUdev is getting the actual info to build up
 // the FILEIO data and match with the enable target
-func (fileio FILEIO) GetFileioUdev(targetCorePath string, fileioNumber string,
-	objectName string) (fio *FILEIO, err error) {
-
-	fileio.Name = "fileio_" + fileioNumber
-	fileio.Fnumber = fileioNumber
-	fileio.ObjectName = objectName
-
+// func (fileio FILEIO) GetFileioUdev(fileioNumber string, objectName string) (fio *FILEIO, err error) {
+func (fs FS) GetFileioUdev(fileioNumber string, objectName string) (*FILEIO, error) {
+	targetCorePath := fs.configfs.Path(TargetCore)
+	fileio := FILEIO{
+		Name:       "fileio_" + fileioNumber,
+		Fnumber:    fileioNumber,
+		ObjectName: objectName,
+	}
 	udevPath := filepath.Join(targetCorePath, fileio.Name, fileio.ObjectName, "udev_path")
```

Comment:
I think this can be simplified a little bit and remove the need for `targetCorePath`.
```golang
	udevPath := fs.configfs.Path(TargetCore, fileio.Name, fileio.ObjectName, "udev_path")
```

Reply:
You are right, I'm rewriting that right now.